### PR TITLE
Add "source" and "conda-forge" URLs for PyPI

### PR DESCRIPTION
**Pull request by maresb**
I had to use Google to find this GitHub project.

### CODE REVIEW

File: setup.py
Comment (shimizukawa) [inline setup.py:44]: What is Conda-Forge?
Sphinx maintainers are not involved in that repository. IMO, this URL cannot be added.
It can be merged once this line is removed.

**Review by shimizukawa — CHANGES_REQUESTED** [review]

File: setup.py
Comment (maresb) [reply setup.py]: Hi! Conda-Forge is a repository similar in spirit to Debian that redistributes open-source software (in accordance with the license, of course). Packages are build to be installed into a "Conda environment" which is the same idea as a Python venv, but it works with and manages dependencies for non-Python software as well.

As requested, I have removed the `Conda-Forge` line.

**Pull request closed without merging**
